src/interactive/AccordionPanelGroup.jsx

```diff
@@ -39,42 +39,54 @@ class AccordionPanelGroup extends React.Component {
 		this.handlePanelClick = this.handlePanelClick.bind(this);
 	}
 
-	static getDerivedStateFromProps(nextProps, prevState) {
-		let derivedPanelListState;
+	// commenting this out as this is not needed for most of the stories in storybook
```

Comment:
Maybe instead of commenting code out, stash it or just save it to another file outside the repo?

Reply:
okay